src/components/ProfilePage/ProfilePage.js

```diff
@@ -0,0 +1,85 @@
+import './profilePage.scss';
+import default_user from '../../assets/images/default-user.jpg';
+import Avatar from './Avatar/Avatar';
+import React, { useEffect, useState, useRef } from 'react';
+import $ from 'jquery';
+
+const ProfilePage = (props) => {
+    const ref_image = React.createRef();
+    const ref_email = useRef(null);
+    const ref_firstName = useRef(null);
+    const ref_lastName = useRef(null);
+    const [user, setUser] = useState([])
+    useEffect(() => {
+        const getUser = async () => {
+            const res = await fetch('http://localhost:9090/users/', {
+                method: 'GET',
+                headers: {
+                    'Authorization': 'Bearer ' + sessionStorage.getItem('jwtToken')
+                }
+            })
+            const data = await res.json();
+            setUser(await data)
+        }
+        getUser()
+    }, [])
+
+    const saveUser = async () => {
+        const emailval = ref_email.current.value.length === 0 || !ref_email.current.value.match(/^([\w.%+-]+)@([\w-]+\.)+([\w]{2,})$/i) ? user.email : ref_email.current.value;
+        const fnameval = ref_firstName.current.value.length === 0 ? user.firstName : ref_firstName.current.value;
+        const lnameval = ref_lastName.current.value.length === 0 ? user.lastName : ref_lastName.current.value;
+        const avatarval = user.avatar;
+        
+        const update_user = {
+            'username': user.username,
+            'email': emailval,
+            'firstName': fnameval,
+            'lastName': lnameval,
+            'avatar': avatarval
+        }
+
+        const res = await fetch('http://localhost:9090/users/update', {
+            method: 'POST',
+            headers: {
+                'Content-type': 'application/json',
+                'Authorization': 'Bearer ' + sessionStorage.getItem('jwtToken')
+            },
+            body: JSON.stringify(update_user),
+        })
+
+        closeForm();
+    }
+    const openFile = () => {
+        $('.file-input').click();
```

Comment:
hmmm, I do not think that we need jquery just for one feature, can you try to find other way?

Reply:
Try this:
http://youmightnotneedjquery.com/#delegate
or this:
https://gomakethings.com/how-to-convert-the-jquery-click-method-to-vanilla-js/

Reply:
JQuery is used not only in this component. Existing of jquery will be fixed after code freeze stage.